<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>288</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>FontAwesome</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="url_line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>98</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>           Url</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="location_line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>138</y>
      <width>331</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>          Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>218</y>
      <width>110</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>138</y>
      <width>71</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>28</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Regular Files ONLY</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
